solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -506,7 +606,18 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         return uint256(10) ** (roundDiv(numOfDigits, length) - 1);
     }
 
-     /**
+    /**
+      * @dev checks if the pool is a 2 reserves / 50%/50% weights pool
+      *
+      * @return true if the pool is standard, false otherwise
+    */
+    function isStandardPool() internal view returns (bool) {

```

Comment:
We cannot use this concept in general, because a pool can alternate between "standard" and "non-standard" (think of v2 pools, for example).
To my understanding, we should set this as an "immutable after activation" state of the pool.
In V1 - it is pretty easy:
- Start it as false
- Set it to true upon adding the 2nd reserve, if both weights are 50%
- Set it to false upon adding the 3rd reserve

Reply:
done